examples/support/observability/README.md

```diff
@@ -1,107 +1,54 @@
-the `keptn-lifecycle-toolkit-system` namespace (
-see <https://prometheus-operator.dev/docs/kube/monitoring-other-namespaces/> as a reference).
-For setting up both Jaeger and Prometheus, please refer to their docs:
-
-* [Jaeger Setup](https://github.com/jaegertracing/jaeger-operator)
-* [Prometheus Operator Setup](https://github.com/prometheus-operator/kube-prometheus/blob/main/docs/customizing.md)
-
-If you don't have an already existing installation of
-Jaeger [manifest](https://github.com/jaegertracing/jaeger-operator/releases/download/v1.38.0/jaeger-operator.yaml) or
-Prometheus, you can run these commands to
-have a basic installation up and running.
-
-```shell
-# Install Jaeger into the observability namespace and the Jaeger resource into the lifecycle-toolkit namespace
-kubectl create namespace observability
-kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/v1.38.0/jaeger-operator.yaml -n observability
-kubectl apply -f config/jaeger.yaml -n keptn-lifecycle-toolkit-system
-
-# Install Prometheus
-kubectl create namespace monitoring
-kubectl apply --server-side -f config/prometheus/setup
-kubectl apply -f config/prometheus/
-```
-
-With these commands, the Jaeger and Prometheus Operator will be installed in the `observability` and `monitoring`
-namespaces, respectively.
-
-## Configuring the OpenTelemetry Collector and Prometheus ServiceMonitor
-
-Once Jaeger and Prometheus are installed, you can deploy and configure the OpenTelemetry collector using the manifests
-in the `config` directory:
-
-```shell
-kubectl apply -f config/otel-collector.yaml -n keptn-lifecycle-toolkit-system
-```
-
-Also, please ensure that the `OTEL_COLLECTOR_URL` env vars of both the `lifecycle-operator`,
-as well as the `keptn-scheduler` deployments are set appropriately.
-By default, they are set to `otel-collector:4317`, which should be the correct value for this tutorial.
-
-Eventually, there should be a pod for the `otel-collector` deployment up and running:
-
-```shell
-$ kubectl get pods -lapp=opentelemetry -n keptn-lifecycle-toolkit-system
-
-NAME                              READY   STATUS    RESTARTS      AGE
-otel-collector-6fc4cc84d6-7hnvp   1/1     Running   0             92m
-```
-
-If you want to extend the OTel Collector configuration to send your telemetry data to other Observability platform, you
-can edit the Collector ConfigMap with the following command:
-
-```shell
-kubectl edit configmap otel-collector-conf -n keptn-lifecycle-toolkit-system
-```
-
-When the `otel-collector` pod is up and running, restart the `keptn-scheduler` and `lifecycle-operator` so they can
-pick up the new configuration.
-
-```shell
-kubectl rollout restart deployment -n keptn-lifecycle-toolkit-system keptn-scheduler lifecycle-operator
-```
+For information about installing and configuring
+the software required, see
+[OpenTelemetry observability](../../../docs/implementing/otel/)
```

Comment:
this is not a website document so we need to provide the full-path to the example
```suggestion
[OpenTelemetry observability](../../../dpcs/content/en/docs/implementing/otel.md)
```